src/core.js

```diff
@@ -45,20 +54,56 @@ class Core extends Model {
 			this.theme = new InfoboxThemeData();
 		}
 
-		// store config for persistence method
-		this.persistOptions = params.persistOptions;
+		let len = routines.length;
+
+		if (len) {
+
+			let bootstrappedRoutines = [];
+
+			for (let i = 0; i < len; i++) {
+				let adapter = routines[i];
+				bootstrappedRoutines.push(Object.assign({}, Serializers[adapter.name], adapter));
```

Comment:
should be `adapter.attributes.name` according to docs and `registerAdapter` below.